packages/blocks/src/api/registration.js

```diff
@@ -131,6 +131,10 @@ import { store as blocksStore } from '../store';
 
 export const serverSideBlockDefinitions = {};
 
+function isObject( object ) {
+	return object !== null && typeof object === 'object';
+}
```

Comment:
On which basis do we decided when to refactor the code "inline" vs creating a `isObject` utility function?

Given the subtlety of this check (which includes the `null` edge case), I'm thinking if it would be better to always refactor the logic to a separate `isObject` function?

Reply:
> On which basis do we decided when to refactor the code "inline" vs creating a isObject utility function?

Depending on the number of usages in a file, essentially. More than a couple of usages makes it easier and less repetitive to have a separate function.

> Given the subtlety of this check (which includes the null edge case), I'm thinking if it would be better to always refactor the logic to a separate isObject function?

I'd disagree, while it adds verbosity, I don't see why we'd always want to keep it a separate function. I think it's enough that we are specifically explicitly excluding the `null` case, and it's a JavaScript quirk, so not much we can do about it.